### DISCUSSION TITLE
Custom smtp configuration issue with email cpanel(devsustain)

### DISCUSSION THREAD

**Discussion by ahmadaliofficial54** [supabase/supabase#1775] [Questions] [answered]
Hi, 

I'm configuring custom SMTP in supabase. I'm using email cpanel(devsustain) smtp. Supabase sending email perfectly fine with devsustain host and generating no error but i didn't get emails on my email address.

I Need help to resolve this issue.

Thanks

**Comment by inian** [comment]
Hi @ahmadaliofficial54, just to confirm are the emails not being sent when using the default SMTP, which uses Supabase's email server or when you use your own SMTP server?

**Reply by ahmadaliofficial54** [reply]
Emails are perfectly working with supabase default SMTP. But its a requirement to configure a custom SMTP. So, when i configured custom SMTP its working and sending emails but i didn't receive email at client side.

**Comment by awalias** [comment, marked as answer]
perhaps they're landing in your spam @ahmadaliofficial54 , or getting blocked by some email delivery services

perhaps you need to setup DMARC on your email provider to increase the likelihood that they won't be picked up by spam detectors. We had to do this on our own provider:

https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-authentication-dmarc.html
